src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+import duke.task.Task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * tasks that happen in a specific time for a duration (e.g. a meeting on 2/10/2019 2-4pm)
+ */
+public class Event extends Task {
+    private LocalDate date;
+
+    public Event(String title, boolean isDone, LocalDate date) {
+        super(title, isDone);
+        this.date = date;
+    }
+
+    public Event(String title, LocalDate time) {
+        this(title, false, time);
+    }
```

Comment:
Would it be better if javadoc comments were included as there is more than one constructor?